packages/fxa-settings/src/components/ScrollToTop/index.tsx

```diff
@@ -30,7 +30,8 @@ export const ScrollToTop = (
     // to it, because reach router can't handle hashes (see reach router
     // issue 32) 🙄
     if (hasHash) {
-      const el = document.querySelector(url.hash);
+      const hashTokens = url.hash.split('?');
```

Comment:
what's this for?

Reply:
Kind of a hack on top of a hack 😕. Using the redirect from above creates a url like

`/settings#connected-services?context=fx_desktop_v3&entrypoint=sendtab&uid=554213....`

Which was causing an error here because `url.hash` included everything after connected-services and was failing to scroll into view. Maybe an alternative is to write the url?